sympy/functions/special/tests/test_spec_polynomials.py

```diff
@@ -160,9 +174,6 @@ def test_assoc_legendre():
 
     assert Plm(n, 0, x) == legendre(n, x)
 
-    raises(ValueError, lambda: Plm(-1, 0, x))
-    raises(ValueError, lambda: Plm(0, 1, x))
```

Comment:
Why did you remove the second assertion here? I think it still raises ValueError, because the current implementation of associate Legendre polynomials insists that abs('2nd index') must be <= '1st index'

Reply:
But the  value error for negative should be removed

Reply:
Done 👍